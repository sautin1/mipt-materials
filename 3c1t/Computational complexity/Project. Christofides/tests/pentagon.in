5
0 0
-1 5
2 7
4 0
5 5