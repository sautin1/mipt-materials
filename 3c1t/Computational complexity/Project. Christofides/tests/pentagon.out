21.40914157811354825